efi/meson.build

```diff
@@ -1,6 +1,25 @@
 generate_sbat = find_program('generate_sbat.py', native: true)
 generate_binary = find_program('generate_binary.py', native: true)
 
+# get source version, falling back to empty
+git = find_program('git', required : false)
+if git.found()
+  source_version = run_command(git, 'describe').stdout().strip()
+else
+  source_version = ''
```

Comment:
as this ends up in the xml, perhaps this is best to be something like "unknown" rather than an empty string?

Reply:
Agreed, fixed.